### DISCUSSION TITLE
function to determine which tables contain a certain variable?

### DISCUSSION THREAD

**Discussion by TSchiefer** [cynkra/dm#1369] [General]
Something like:
```
names(<dm>)[map_lgl(dm_get_tables(<dm>), ~ any(grepl("<colname>", colnames(.))))]
```
Might make sense if you have a dm with a lot of tables.

Not sure if necessary.

**Comment by TSchiefer** [comment]
maybe #329 would mostly solve this already

**Comment by krlmlr** [comment]
A lot of that will be solved by `dm_meta()` ?

**Reply by TSchiefer** [reply]
Oh alright, wasn't aware that `dm_meta()` is supposed to be exported. If that's the case, we can close this discussion as well as #329.

**Reply by krlmlr** [reply]
Yeah, we should offer a way to extract a `dm_meta()` compatible data structure from a dm object. Let's make this the scope of #329.

Not sure how to close discussions. We can upvote contributions perhaps.
